saleor/graphql/campaign/enums.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+
+
+class IndustryEnum(Enum):
```

Comment:
Industries need to be values from this endpoint https://pkb.web-production.vonq-aws.com/products/industries/
The labels are in the `name` key. The raw values passed to the order are in the `id` key